playbooks/files/rax-maas/plugins/octavia_api_local_check.py

```diff
@@ -32,14 +33,20 @@ def check(auth_ref, args):
     OCTAVIA_ENDPOINT = 'http://{ip}:9876/v1'.format(ip=args.ip,)
```

Comment:
This has "/v1" in it.
Ugh, still using the old port # way to access APIs? We should consider moving to the wsgi non-port based deployment.